safehttp/plugins/htmlinject/htmlinject_test.go

```diff
@@ -15,10 +15,10 @@
 package htmlinject
 
 import (
-	"html/template"
 	"os"
 	"strings"
 	"testing"
+	"text/template"
```

Comment:
Why?

Reply:
Linters complain when they spot usage of `html/template`

Reply:
Please keep it. Linters are here to help, but if they don't, we ignore them. Changing this to `text/template` makes it confusing, as this is a `htmlinject_test` file that's supposed to work with HTML.